tests/integration/test_charm.py

```diff
@@ -28,17 +30,67 @@ async def test_flask_is_up(
         assert "Hello, World!" in response.text
 
 
-@pytest.mark.parametrize("update_config", [{"webserver_timeout": 7}], indirect=True)
+@pytest.mark.parametrize(
+    "update_config, timeout",
+    [({"webserver_timeout": 7}, 7), ({"webserver_timeout": 5}, 5), ({"webserver_timeout": 3}, 3)],
+    indirect=["update_config"],
+    ids=["timeout-7", "timeout-5", "timeout-3"],
```

Comment:
Do you think it would be clearer to put the `id`s with the definition for each test? E.g.:

```python
[pytest.param({"webserver_timeout": 7}, 7, id="timeout-7"), ...]
```

Reply:
Doesn't know that before, updated, thanks!